plainly-plugin/src/ui/components/projects/ProjectsList.tsx

```diff
@@ -0,0 +1,66 @@
+import { useGetProjects } from '@src/ui/hooks';
+import { useProjectData } from '@src/ui/hooks/useProjectData';
+import { Routes } from '@src/ui/types';
```

Comment:
* I think the selected one should be out of the list at the top somehow
* With clear selection, maybe some kind of the green ring applied
* Also more data with this one, for example I want to have local and remote revision, maybe templates could and then set of actions maybe as buttons in the bottom of the card
* yes should be some kind of card rather that a single row

Maybe we can discuss this quickly? :call_me_hand: 

![image](https://github.com/user-attachments/assets/5926fe6e-dbca-482d-9683-527b8b23fac2)